Core/GDCore/Project/Project.h

```diff
@@ -1,1002 +1,1002 @@
-  gd::String scaleMode;
-  bool adaptGameResolutionAtRuntime;  ///< Should the game resolution be adapted
-                                      ///< to the window size at runtime
-  gd::String
-      sizeOnStartupMode;  ///< How to adapt the game size to the screen. Can be
-                          ///< "adaptWidth", "adaptHeight" or empty
-  std::vector<std::unique_ptr<gd::Layout> > scenes;  ///< List of all scenes
-  gd::VariablesContainer variables;  ///< Initial global variables
-  std::vector<std::unique_ptr<gd::ExternalLayout> >
-      externalLayouts;  ///< List of all externals layouts
-#if defined(GD_IDE_ONLY)
-  std::vector<std::unique_ptr<gd::EventsFunctionsExtension> >
-      eventsFunctionsExtensions;
-#endif
-  gd::ResourcesManager
-      resourcesManager;  ///< Contains all resources used by the project
-  std::shared_ptr<gd::ImageManager>
-      imageManager;  ///< Image manager is accessed thanks to a (smart) ptr as
-                     ///< it can be shared with GD C++ Platform projects.
-  std::vector<gd::String> extensionsUsed;  ///< List of extensions used
-  std::vector<gd::Platform*>
-      platforms;  ///< Pointers to the platforms this project supports.
-  gd::String firstLayout;
-#if defined(GD_IDE_ONLY)
-  bool useExternalSourceFiles;  ///< True if game used external source files.
-  std::vector<std::unique_ptr<gd::SourceFile> >
-      externalSourceFiles;  ///< List of external source files used.
-  gd::String author;        ///< Game author name
-  gd::String packageName;   ///< Game package name
-  gd::String orientation;   ///< Lock game orientation (on mobile devices).
-                            ///< "default", "landscape" or "portrait".
-  bool
-      folderProject;  ///< True if folder project, false if single file project.
-  gd::String gameFile;                    ///< File of the game
-  gd::String latestCompilationDirectory;  ///< File of the game
-  gd::Platform*
-      currentPlatform;  ///< The platform being used to edit the project.
-  gd::PlatformSpecificAssets platformSpecificAssets;
-  gd::LoadingScreen loadingScreen;
-  std::vector<std::unique_ptr<gd::ExternalEvents> >
-      externalEvents;  ///< List of all externals events
-  ExtensionProperties
-      extensionProperties;              ///< The properties of the extensions.
-  mutable unsigned int gdMajorVersion;  ///< The GD major version used the last
-                                        ///< time the project was saved.
-  mutable unsigned int gdMinorVersion;  ///< The GD minor version used the last
-                                        ///< time the project was saved.
-  mutable unsigned int gdBuildVersion;  ///< The GD build version used the last
-                                        ///< time the project was saved.
-  mutable bool dirty;  ///< True to flag the project as being modified.
-#endif
-};
-
-}  // namespace gd
-
-#endif  // GDCORE_PROJECT_H
+/*
```

Comment:
Seems like there are no changes in Project.cpp/Project.h - can you revert these to clean up the history and ease the review?